src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -1,5 +1,5 @@
 import type { AssetId } from '@shapeshiftoss/caip'
-import { reactQueries } from 'react-queries'
+import { thornode } from 'react-queries/queries/thornode'
```

Comment:
This is the only change which could be problematic. Required to break dependency cycle. Seems to work fine